modules/tinymce/src/plugins/image/main/ts/ui/Dialog.ts

```diff
@@ -335,13 +327,13 @@ const closeHandler = (state: ImageDialogState) => () => {
 };
 
 const makeDialogBody = (info: ImageDialogInfo) => {
-  if (info.hasAdvTab || info.hasUploadUrl || info.hasUploadHandler) {
+  if (info.hasAdvTab || info.hasUploadUrl) {
```

Comment:
This will break things, as we need to only show the upload tab if either the `images_upload_url` or `images_upload_handler` is configured.

- [ ] Revert this change please, as we still need this check

Reply:
Done.